package.json

```diff
@@ -4,6 +4,11 @@
   "description": "This is a fairly functional work-in-progress. Let's try to keep it clean and functionally minimal as we evolve it.",
   "main": "content.js",
   "scripts": {
+    "clean": "rimraf dist",
+    "predev": "npm run clean",
+    "dev": "cross-env NODE_ENV=development parcel watch src/content.js src/hotReload.js ",
```

Comment:
`parcel watch` is used instead of the default development mode of just `parcel` because there is no need to be running a web server as this is a browser extension. That is the only difference between the two commands.